apps/web-giddh/src/app/signup/signup.component.ts

```diff
@@ -78,9 +86,12 @@ export class SignupComponent implements OnInit, OnDestroy {
         private loaderService: LoaderService,
         private toaster: ToasterService,
         private authenticationService: AuthenticationService,
-        private ngZone: NgZone
+        private ngZone: NgZone,
+        @Inject(ServiceConfig) private serviceConfig,
+        private generalService : GeneralService
     ) {
-        this.urlPath = isElectron ? "" : AppUrl + APP_FOLDER;
+        this.urlPath = isElectron ? "" : (this.serviceConfig.AppUrl || AppUrl) + APP_FOLDER;
+        this.giddhDomainUrl = this.serviceConfig.AppUrl || 'https://giddh.com';
```

Comment:
```suggestion
        this.giddhDomainUrl = this.serviceConfig.AppUrl || DOMAIN_URL;
```

DOMAIN_URL : 'https://giddh.com' (make constant and use every where)